templates/channel.pug

```diff
@@ -43,6 +43,7 @@ html(lang="en")
                 i#userlisttoggle.glyphicon.glyphicon-chevron-down.pull-left.pointer(title="Show/Hide Userlist")
                 span#usercount.pointer Not Connected
                 span#modflair.label.label-default.pull-right.pointer Name Color
+                span#notifications.label.label-default.pull-right.pointer Notifications
```

Comment:
Per my below comment about the setting, I think this can be removed since it creates confusion to configure the same setting in multiple places, and I think this setting is one that doesn't need to be quick-toggleable.

Reply:
My motivation for that was that I often leave the stream in the background, an and option to hide notifications was pretty handy when I was not actively participating in the chat and didn't particularly care about what was being written there at that moment.
Perhaps it should be a separate setting (mute all kinds/selected kinds of notifications, like a sort of do-not-disturb mode), or maybe it should cycle the setting through the three values from the comment above, or perhaps that's going too far...

Reply:
The join message is a chat message, so it obeys the same notification policy.  That said, you shouldn't be receiving a join message for yourself -- so I think that should only occur if an IP with your name as an alias, or with your name as a substring of their name (due to that issue I've never fixed) joins.

Reply:
Oh that's probably it. I'm testing on localhost in a second private window.